client/components/site-verticals-suggestion-search/index.jsx

```diff
@@ -31,10 +31,8 @@ export class SiteVerticalsSuggestionSearch extends Component {
 		lastUpdated: PropTypes.number,
```

Comment:
Don't need this prop any more. It was a property of `getHttpData` 👍

Reply:
Great catch. Removed in 44c7742.